spec/descriptions/createSyntheticTest.md

```diff
@@ -0,0 +1,55 @@
+This API endpoint creates a Synthetic Test.
+
+## A note on testFrequency parameter:
+The unit of the testFrequency parameter is minute. 
+
+## Sample script and payload: 
+- A sample script to create a simple Ping API test
+
+```json
+curl -k -v -X POST \
+http://<hostname>/api/synthetics/settings/tests \
+-H 'authorization: apiToken valid-api-token' \
+-H 'content-type: application/json' \
+-d '{
+    "label":"Test_SimplePing",
```

Comment:
not knowing exactly, but from structure it looks like this can also go into spec/playload/createSyntheticTest.json

Reply:
We may have multiple sample payloads due to that Synthetics have different test types. Is it OK to put all payloads in spec/payload/createSyntheticTest.json? Thanks.